public/data/job-openings-data.json

```diff
@@ -108,6 +108,49 @@
           "experience": "0-2 years"
         }
       ]
+    },
+    {
+      "id": 4,
+      "companyName": "MagnusMinds IT Solution LLP",
+      "location": "Ahmedabad, India",
+      "bio": "MagnusMinds IT Solution is a leading IT Solution provider Company in India. We provide flexible business strategies with creative tools and advanced methodologies to help businesses achieve success. Our rich & varied experience of 15+ years and stringent quality standards ensure we develop software solutions that give your business an edge over competitors. MagnusMinds is looking for some of the talented individuals, who have an interest in solving real-world business problems and challenges through web-based solutions.",
+      "companyLogoUrl": "https://drive.google.com/file/d/1DTFL3mvGzOAG6WKZlHE3THFxmpYNfRNh/view?usp=share_link",
```

Comment:
Hello @mkinjal203 
Please share proper URL in companyLogoUrl. You can refer other objects from the JSON.

Reply:
Hello @JaldeepAsodariya
I have updated companyLogoUrl with proper link. Please verify.